apps/image-focal-point/package.json

```diff
@@ -19,7 +20,8 @@
     "start": "contentful-extension-scripts start --serve-only",
     "build": "contentful-extension-scripts build --no-inline",
     "lint": "eslint ./ --ext .js,.jsx,.ts,.tsx",
-    "deploy": "aws s3 sync ./build ${STATIC_S3_BASE}/focal-point --acl public-read",
+    "deploy": "contentful-app-scripts upload --ci --bundle-dir ./build --organization-id ${DEFINITIONS_ORG_ID} --definition-id 342Q2DqCjmsdN5BJCEPkrJ --token ${CONTENTFUL_CMA_TOKEN}",
+    "deploy:test": "contentful-app-scripts upload --ci --bundle-dir ./build --organization-id ${TEST_ORG_ID} --definition-id ${TEST_APP_ID} --token ${CONTENTFUL_CMA_TOKEN}",
```

Comment:
my assumption here is that we'll create a new definition which we can just use to test-deploy all the apps - since it seems that the main thing we're testing here is that the upload / activation will succeed

Reply:
Makes sense